src/core/include/cesium/omniverse/FabricMaterial.h

```diff
@@ -82,6 +82,7 @@ class FabricMaterial {
     void createTexture(const omni::fabric::Path& path);
     void createImageryLayer(const omni::fabric::Path& path);
     void createImageryLayerResolver(const omni::fabric::Path& path, uint64_t textureCount);
+    void createPolygonImageryLayerResolver(const omni::fabric::Path& path, uint64_t textureCount);
```

Comment:
Rename to `createClippingImageryLayerResolver`

Reply:
There are a few other areas in `FabricMaterial` and `cesium.mdl` that need to be renamed. Make sure to do an exhaustive search for those.

Reply:
Changed, also in the tokens.

There's now an "imagery layer resolver" and "clipping imagery layer resolver". I think that's OK, though, since overlaying imagery is by far the default.